### ISSUE TITLE
Accessibility audit for Blog

### ISSUE THREAD

**Issue by mattjmcnaughton** [mattjmcnaughton/blog#51]
*Issue*
I want this blog to be accessible to as many people as possible. I have not yet performed an audit of its accessibility, so there may be issues that I'm missing.

*Implementation*
See guidance in http://www.karlgroves.com/2013/09/05/the-6-simplest-web-accessibility-tests-anyone-can-do/.

*Definition of Done*
- [ ] Identify a measurable accessibility standard I want to meet, and update my blog until it reaches that standard.
- [ ] Blog post.
